lib/node_modules/@stdlib/_tools/scaffold/math-iter-unary/scripts/scaffold.sh

```diff
@@ -41,19 +41,19 @@ alias=${ALIAS:-'iterTest'}
 
 # Define the package description:
 # shellcheck disable=SC2016
-pkg_desc=${PKG_DESC:-'Create an iterator which computes `gamma(x+1) - 1` for each iterated value.'}
+pkg_desc="Create an iterator which ${DESC:-'computes \`gamma(x+1) - 1\`'} for each iterated value."
```

Comment:
```suggestion
pkg_desc="Create an iterator which ${DESC:-'TODO'} for each iterated value."
```

Let's add a placeholder.